### PR TITLE
Updated NSIS keywords (in-sync with NSIS revision 7154)

### PR DESCRIPTION
**Pull request by negrutiu**

### CODE REVIEW

File: PowerEditor/src/langs.model.xml [hunk 1]
```diff
@@ -239,9 +239,9 @@
...
...4 MB_ICONEXCLAMATION MB_ICONINFORMATION MB_ICONQUESTION MB_ICONSTOP MB_OK MB_OKCANCEL MB_RETRYCANCEL MB_RIGHT MB_RTLREADING MB_SETFOREGROUND MB_TOPMOST MB_USERICON MB_YESNO MB_YESNOCANCEL NOCUSTOM NOERRORS NORMAL NoBOM NoWorkingDir OEM OFFLINE OVERWRITE READONLY REGEDIT5 REPLACE RESET RESIZETOFIT RESIZETOFITHEIGHT RESIZETOFITWIDTH RawNL SET SHCTX SHCTX32 SHCTX64 SHCTXANY SHELL_CONTEXT SHORT SILENT SOLID STRIKE STRINGID SW_HIDE SW_SHOW SW_SHOWDEFAULT SW_SHOWMAXIMIZED SW_SHOWMINIMIZED SW_SHOWMINNOACTIVE SW_SHOWNA SW_SHOWNORMAL SYSTEM TEMPORARY TIMEOUT UNDERLINE UTF16BE UTF16BOM UTF16LE UTF8 UTF8SIG WINDOW WINDOWTEXT admin all alwaysoff amd64 ansi arm64 auto both bottom bzip2 code colored components current custom default directory disable enable error explorer false fileexists force gray grey hide highest ifdiff ifempty ifnewer instfiles lastused left license listonly lzma manual nevershow noerrors none nonfatal normal notext notset off on oname packed permonitor plugin pop push right show silent silentlog smooth system textonly top true try un.components un.custom un.directory un.instfiles un.license un.uninstConfirm unicode uninstConfirm user warning windows x64 x86 zlib</Keywords>
```
Comment (donho) [inline PowerEditor/src/langs.model.xml:244]: Just check **3DFACE** and it doesn't seem the keyword of NSIS.
Could you provide any link or ref of NSIS keywords?

**Comment by negrutiu** [comment]
NSIS source code reference:
[nsis/Source/script.cpp@72](https://github.com/kichik/nsis/blob/c88c1f54ad08189e3d47bd9b40db4a61a8a0b424/Source/script.cpp#L72)

NSIS syntax example:
```
SetCtlColors $mui.InstFilesPage.Text SYSCLR:3DFACE SYSCLR:HIGHLIGHT
```

**Comment by sasumner** [comment]
@negrutiu That's just some sample code that uses it.  What was asked for was a NSIS keyword reference document.

**Comment by negrutiu** [comment]
The official NSIS scripting reference is here: https://nsis.sourceforge.io/Docs/Chapter4.html
NOTE: SetCtlColors in documented in chapter 4.9.14.16

The documentation is quite extensive, yet a few (legitimate) keywords are not mentioned there.
It happens that "3DFACE" is one of them, although it is real and it works if used in a script. I've provided you earlier with a working example.

My list of keywords was extracted directly from the NSIS source code, which I'm very familiar with.
These are all instructions/attributes/keywords that the NSIS compiler is looking for when parsing a script...

Sorry guys, that's pretty much all I have.
Some of the keywords are (not yet) documented. I can't provide you with documentation references for them...

**Comment by sasumner** [comment]
@negrutiu Your explanation makes sense.  I'm sure you can understand the reluctance to insert something like this into a product without a formal reference.  Note that I'm not saying it won't be done. :-)

**Comment by negrutiu** [comment]
Of course I understand. I would do the same ;)

**Comment by donho** [comment]
@negrutiu 
Of course that the undocumented keywords won't be accepted.
Some of keywords you added (like 3DFACE ACP BRANDING) cannot be found in the link you provided. I cannot even find their relation with NSIS by googling.

**Pull request closed without merging**